pages/robot/tabs/components/index.php

```diff
@@ -496,16 +380,18 @@ function render_components(data) {
                             </div>
                             <div class="modal-body">
                                 <p id="{description_id}" class="text-left"></p>
+                                <br/>
                                 <div class="row">
-                                    <div class="col-sm-2">
+                                    <div class="col-md-2">
                                         <p>Also, you could:</p>
                                     </div>
-                                    <div class="col-md-4 bg-light text-left">
-                                        <button type="button" class="btn btn-white btn-custom-sm text-left" id="{btn_id_logs_node}">Download the log file for this ROS Node</button>
+                                    <div class="col-md-3 bg-light text-left">
+                                        <button type="button" class="btn btn-sm text-left" id="{btn_id_logs_node}">Download this ROS node logs</button>
```

Comment:
Can we do "Download logs for this ROS node", or "Download this ROS node's logs"?